### PR TITLE
Add et.js.coffee

### PR DESCRIPTION
**Pull request by holgerkilumets**

### CODE REVIEW

File: app/assets/javascripts/uploadcare/locale/et.js.coffee [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            done: 'Valmis'
+      footer:
+        text: 'powered by'
+        link: 'uploadcare'
+
+
+# Pluralization rules taken from:
+# http://unicode.org/repos/cldr-tmp/trunk/diff/supplemental/language_plural_rules.html
+uploadcare.namespace 'locale.pluralize', (ns) ->
+  ns.en = (n) ->
```
Comment (dmitry-mukhin) [inline app/assets/javascripts/uploadcare/locale/et.js.coffee]: should be `ns.et`

**Comment by dmitry-mukhin** [comment]
You should have created a copy of the file, not rename it :)
Nevermind, we'll handle that.

**Comment by holgerkilumets** [comment]
Oops, sorry! Not a developer and new to github. Hope it's not too much trouble, thanks!

**Comment by dmitry-mukhin** [comment]
no probs, thanks and don't forget the [this one](https://github.com/uploadcare/social-l10n)

**Pull request closed without merging**